code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -7,28 +7,25 @@
 ///returns a list of "damtype" => damage description based off of which bodypart description is most common
 ///used in human examines
 /mob/living/carbon/human/proc/get_majority_bodypart_damage_desc()
-	var/most_seen_brute = 0
-	var/most_seen_burn = 0
-	var/brute_desc = ""
-	var/burn_desc = ""
-	var/list/seen_brute = list()
-	var/list/seen_burn = list()
+	var/list/seen_damage = list() // This looks like: ({Damage type} = list({Damage description for that damage type} = {number of times it has appeared}, ...), ...)
+	var/list/most_seen_damage = list() // This looks like: ({Damage type} = {Frequency of the most common description}, ...)
+	var/list/final_descriptions = list() // This looks like: ({Damage type} = {Most common damage description for that type}, ...)
 	for(var/obj/item/bodypart/part as anything in bodyparts)
-		//brute
-		if(!seen_brute[part.brute_damage_desc])
-			seen_brute[part.brute_damage_desc] = 1
-		else
-			seen_brute[part.brute_damage_desc] += 1
-		if(seen_brute[part.brute_damage_desc] > most_seen_brute)
-			brute_desc = part.brute_damage_desc
-		//burn
-		if(!seen_burn[part.burn_damage_desc])
-			seen_burn[part.burn_damage_desc] = 1
-		else
-			seen_burn[part.burn_damage_desc] += 1
-		if(seen_burn[part.burn_damage_desc] > most_seen_burn)
-			burn_desc = part.burn_damage_desc
-	return list(BRUTE = brute_desc, BURN = burn_desc)
+		for(var/i in 1 to part.damage_examines.len)
```

Comment:
Simple in loop here, you don't really need the index ? ```for(var/damage_type in part.damage_examines)```

Reply:
I need the index (aka, damage type) to compare the damage descriptions to only the damage descriptions of that same type.

Reply:
you should still be able to make this comparison, I'm confused, it's just saving a line

Reply:
> you should still be able to make this comparison, I'm confused, it's just saving a line

It's entirely possible that I'm being thick in this moment, but how exactly would I make the comparison without having the index?

Reply:
likewise, I may be too. I don't see anywhere else you need the index. The code could be 
```
for(var/damage_type in part.damage_examines)
    var/damage_desc = part.damage_examines[damage_type]
```

Reply:
> likewise, I may be too. I don't see anywhere else you need the index. The code could be
> 
> ```
> for(var/damage_type in part.damage_examines)
>     var/damage_desc = part.damage_examines[damage_type]
> ```

Huh, I thought that would cycle through the description texts themselves rather the indices.
Turns out, I am an idiot.
Thank you.